packages/@ama-sdk/generator-sdk/src/generators/java-mdk-core/templates/swagger-codegen-mdk/README.md

```diff
@@ -13,22 +13,22 @@ First compile this project, run:
 mvn package
 ```
 
-You can configure the generation of the MDK code generation thanks to the following file: `\config/swagger-codegen-resteasy-config.json` 
+You can configure the generation of the MDK code generation thanks to the following file: `\config/swagger-codegen-resteasy-config.json`
 
 ```
 {
-  "groupId" : "com.airline.dapi",                   // Generated Maven project Goupt ID 
+  "groupId" : "com.airline.dapi",                   // Generated Maven project Goupt ID
```

Comment:
do we want all of these references to dapi ?

Reply:
this file is already present twice in the repo. I created an issue to remove all references to DAPI/DES/Digital https://github.com/AmadeusITGroup/otter/issues/1190